docs/rpc-api.md

```diff
@@ -62,18 +62,20 @@ Below is a table of provided methods.
 | [eth_chainId](https://besu.hyperledger.org/en/stable/Reference/API-Methods/#eth_chainid)                                                  | [Chain_ID](../README.md#configuration)    | Relay Only    |
 | [eth_coinbase](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_coinbase)                                                       | `-32601`                                  | N/A   |
 | [eth_estimateGas](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_estimategas)                                                 | N/A                                       | Both Nodes  |
```

Comment:
eth_estimateGas returns a hardcoded hex of one of: 
- for contract function executions: `400_000`
- for transfer transactions: `21_000`
- for transfer transactions that trigger a hollow account creation: `587_000`

It makes a request to the Mirror node only.

Reply:
Good points. Updated static values but leaving source as N/A as that's just for determining whether it's a transfer or an execution.
Once HIP 584 is implemented this will turn to `Mirror Node`